Makefile

```diff
@@ -31,9 +32,11 @@ install-linters: ## Install linters
 
 lint: ## Run linters. Use make install-linters first
 	${OPTS} golangci-lint run -c .golangci.yml ./...
-	# The govet version in golangci-lint is out of date and has spurious warnings, run it separately
-	${OPTS} go vet -all ./...
+	# The govet version in golangci-lint is out of date and has spurious warnings
```

Comment:
Is there a reason for removing `${OPTS} go vet -all ./...`?

Reply:
Not at all. Removed it unknowingly...